l10n_es_mis_report/data/mis_report_balance_normal.xml

```diff
@@ -0,0 +1,2473 @@
+            <field name="name">es12520</field>
+            <field name="description">2. Créditos a empresas</field>
+            <field name="expression">
+                +bale[5325%,5345%,542%,543%,547%,5955%,598%]
+            </field>
+            <field name="auto_expand_accounts">True</field>
+            <field
+                name="auto_expand_accounts_style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l4i"
+            />
+            <field
+                name="style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l4"
+            />
+        </record>
+        <record id="mis_report_kpi_es_balance_normal_12530" model="mis.report.kpi">
+            <field name="report_id" ref="mis_report_es_balance_normal" />
+            <field name="type">num</field>
+            <field name="compare_method">pct</field>
+            <field name="sequence">660</field>
+            <field name="name">es12530</field>
+            <field name="description">3. Valores representativos de deuda</field>
+            <field name="expression">+bale[5315%,5335%,541%,546%,5945%,597%]</field>
+            <field name="auto_expand_accounts">True</field>
+            <field
+                name="auto_expand_accounts_style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l4i"
+            />
+            <field
+                name="style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l4"
+            />
+        </record>
+        <record id="mis_report_kpi_es_balance_normal_12540" model="mis.report.kpi">
+            <field name="report_id" ref="mis_report_es_balance_normal" />
+            <field name="type">num</field>
+            <field name="compare_method">pct</field>
+            <field name="sequence">670</field>
+            <field name="name">es12540</field>
+            <field name="description">4. Derivados</field>
+            <field name="expression">+bale[5590%,5593%]</field>
+            <field name="auto_expand_accounts">True</field>
+            <field
+                name="auto_expand_accounts_style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l4i"
+            />
+            <field
+                name="style_id"
+                ref="l10n_es_mis_report.mis_report_style_l10n_es_l4"
+            />
+        </record>
+        <record id="mis_report_kpi_es_balance_normal_12550" model="mis.report.kpi">
+            <field name="report_id" ref="mis_report_es_balance_normal" />
+            <field name="type">num</field>
+            <field name="compare_method">pct</field>
+            <field name="sequence">680</field>
+            <field name="name">es12550</field>
+            <field name="description">5. Otros activos financieros</field>
+            <field name="expression">
+                +bale[5355%,545%,548%,565%,566%] + pbale[550%] + pbale[551%] +
+                pbale[554%] + pbale[5525%]
```

Comment:
This line should be in this form
+bale[5355%,545%,548%,565%,566%] +pbale[550%] +pbale[551%] +pbale[554%] +pbale[5525%]